cli/runSentryRelease.sh

```diff
@@ -55,6 +55,9 @@ function options() {
     # Parse options
     while getopts ":hm:p:r:su:" opt; do
```

Comment:
I don't see a corresponding changes to getopts?

Reply:
Fixed